internal/query/roles_v2.go

```diff
@@ -426,7 +430,10 @@ func (e *engine) DeleteRoleV2(ctx context.Context, roleResource types.Resource)
 		errs = append(errs, err)
 	}
 
-	// 2.c TODO remove all role relationships in role bindings associated with this role
+	// 2.c remove all role relationships in role bindings associated with this role
+	if err := e.deleteRoleBindingsForRole(ctx, roleResource); err != nil {
+		errs = append(errs, err)
+	}
```

Comment:
Why do this? It seems like it would make more sense to disallow deleting a role unless all of its bindings are deleted first.